web/ui/src/lib/clustersMap/campus/seoul.ts

```diff
@@ -0,0 +1,200 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { CampusNames, ICampus } from '../types';
+
+//
+export class Seoul extends Campus implements ICampus {
+  emoji = (): string => '🇰🇷';
+
+  name = (): CampusNames => 'seoul';
+
+  extractor = (identifier: string) => {
```

Comment:
I just put an update on main branch and replace this method with another (sorry for that). 
Can you update your branch ?